src/main/java/org/generationcp/middleware/service/impl/study/advance/AdvanceServiceImpl.java

```diff
@@ -691,4 +750,11 @@ private MeasurementVariable getObservationUnitVariable(final DatasetDTO datasetD
 		return null;
 	}
 
+	private PedigreeService getPedigreeService() {
```

Comment:
Is getPedigreeService necessary? I think the pedigreeService is already injected using `@Resource`